tests/scenario/test_machine_charm/test_multiple_subordinates.py

```diff
@@ -2,13 +2,28 @@
 # See LICENSE file for licensing details.
 
 import json
+from unittest.mock import PropertyMock, patch
 
 import charm
+import pytest
 from scenario import Context, PeerRelation, State, SubordinateRelation
 
 from tests.scenario.helpers import get_charm_meta
 
 
+@pytest.fixture(autouse=True)
+def mock_config_path(placeholder_cfg_path):
```

Comment:
instead of repeating this from the test_relation_status module, should we add a `context` fixture to conftest.py, and add these autouse patches on top of it so they're used uniformly across the tests?